Code/Core/FileIO/PathUtils.cpp

```diff
@@ -290,4 +290,39 @@
     outRelativeFileName += pathB;
 }
 
+// GetBaseName
 //------------------------------------------------------------------------------
+
+/* static */ void PathUtils::GetBaseName( const AString& pathName, AString& baseName )
+{
+    const char* slashPos = nullptr;
+    if ( nullptr != ( slashPos = pathName.FindLast( NATIVE_SLASH ) ) )
+        baseName = slashPos + 1;
+    else
+        baseName = pathName.Get();
```

Comment:
Both baseName and pathName are AStrings, so this can be:
```
basetName = pathName;
```

Reply:
Fixed.